experiments/remote_execute_api.yaml

```diff
@@ -0,0 +1,14 @@
+experiments:
+  - metadata:
+      name: remote-execute-api
+      type: remote_execute-api
+      namespace: default
+    parameters:
+      image: alconen/egress_server
+      target_port: 4000
+      path: /experiment/CheckEgress/
```

Comment:
I would actually keep the container's URL path and expected body/response constant and not configurable in the experiment, instead making the called URLs and their response codes configurable